src/site/lab10.rst

```diff
@@ -0,0 +1,165 @@
+*******
+Lab #10
+*******
+
+* Refer to the topic pages for this lab
+    * You will find many of the answers to your questions there
+* Feel free to use your laptop if you have it
+* I strongly encourage you to work with others in the lab
+* Ensure I have recorded your completion. Failure to do so will result in a grade of 0
+* When you get stuck, do me a favour and ask those sitting around you for help
+    * I want people to get used to working together in the labs
+    * Peer teaching and peer learning is super effective
+
+.. note::
+
+    This lab is making use of integer arrays for sorting, but if you would like, feel free to use ``IndexedBags``. If
+    choose to do this, here's a tip for your method declarations:
+
+    ``public static <T extends Comparable<T>> void bubbleSort(IndexedBag<T> toSort)``
+
+    Also remember to use ``.compareTo``.
+
+
+Starting Point
+==============
+
+This lab focuses on the sorts discussed in :doc:`topic 25 </topic25>`.
+
+Below is some code to start from. You will need to ``import java.util.Random;``.
+
+.. code-block:: java
+
+    public static void main(String[] args){
+
+        int[] toSort = createUnsortedArray(10000, 10);
+        printArray(toSort);
+
+        double startTime = System.nanoTime();
+        bubbleSort(toSort);
+        //insertionSort(toSort);
+        //selectionSort(toSort);
+        double endTime = System.nanoTime();
+
+        printArray(toSort);
+        System.out.println((endTime - startTime)/1000000);  // Runtime in ms
```

Comment:
Move the comment into the string being output instead. `println("runtime: " + (x) + " ms")`